omero/sysadmins/windows-migration.rst

```diff
@@ -1,65 +1,4 @@
 Migration from OMERO 5.2 on Windows to OMERO 5.3 on Linux
 =========================================================
```

Comment:
Can we just get rid of this page? Or is this kind of like an API deprecation?

Reply:
I defer to @joshmoore here. Although since the page has only existed since 5.3 maybe that adds weight to the argument we could delete it altogether.

Reply:
5.4.0 allows breaking change. :smiley:

Reply:
I guess we could also point people at the page when we announce that they must upgrade from 5.2

Reply:
Is there an statement somewhere to say, "if you are updating from a version not listed on this page, then please see `|LASTVERSION|` documentation" ? If so, I'd assume that suffices.

Reply:
There is both that AND I've added a specific link to this page underneath that statement. In that case, I'll go ahead and delete it.